### DISCUSSION TITLE
能不能升级版本不要总是更改底层字段和结构

### DISCUSSION THREAD

**Discussion by OrageKK** [vuepress-theme-hope/vuepress-theme-hope#4196] [Q&A] [answered]
一升级就报错，修复要查找好久

**Comment by Mister-Hope** [comment]
我们最近又更改了什么底层字段和结构？

现在发布破坏性更改的频率应该是一个月一次，而且都应该有明确的目标。

**Reply by OrageKK** [reply]
嗯，是我太久没升级了吧，自定义内容多，引用很多底层的东西。每次升级都有各种报错

**Reply by Mister-Hope** [reply, marked as answer]
> 嗯，是我太久没升级了吧，自定义内容多，引用很多底层的东西。每次升级都有各种报错

我们完全不保证内部的组件结构、属性以及类名保持一致。对外的接口应该是统一的。

**Reply by OrageKK** [reply]
> > 嗯，是我太久没升级了吧，自定义内容多，引用很多底层的东西。每次升级都有各种报错
> 
> 我们完全不保证内部的插件结构、属性以及类名保持一致。对外的接口应该是统一的。

好的，单纯对外使用的接口确实没有太大的变化，是我自定义的东西比较多，我指的是比如内部使用的类名，全局变量等的变更。正式版之后应该会趋于稳定吧

**Reply by Mister-Hope** [reply]
对，功能基本已经做完了。
